src/content/docs/en/install/manual.mdx

```diff
@@ -59,6 +59,10 @@ Once you are in your new directory, create your project `package.json` file. Thi
 
 First, install the Astro project dependencies inside your project.
 
+:::note[Important]
```

Comment:
I do like the note here. I think this is where people are installing manually, and are most likely to make this "mistake"!